src/sparse_voxels.jl

```diff
@@ -62,8 +58,9 @@ function SparseVoxelGrid{T <: AbstractVector}(points::Vector{T}, voxel_size)
     # Allocate ranges for the indices of points in each voxel based on the counts
     voxel_info = Dict{VoxelId, UnitRange{Int}}()
     current_index = 1
-    for (group_id, group_size) in group_counts
-        voxel_info[group_id] = current_index:current_index+group_size-1
+    for group_id in keys(group_counts)
```

Comment:
I'll check inference on 0.5.2, if this is still a problem, let's add `TODO: Using keys(group_counts) is a workaround for inference in julia-0.5 - let's just iterate over the dictionary in the future`